src/test/java/org/mule/module/apikit/model/EntityModelParserTestCase.java

```diff
@@ -49,4 +51,21 @@ public void testPositive3() throws IOException, EntityModelParsingException {
     Assert.assertEquals("ordersId", entities.get(1).getIdElementName());
   }
 
+  @Test
+  public void testNegative() throws IOException, EntityModelParsingException {
```

Comment:
why testNegative and testNegative2 are differents ?

Reply:
Renamed